bitnami/external-dns/templates/psp-clusterrole.yaml

```
@@ -17,5 +17,5 @@ rules:
   resources: ['podsecuritypolicies']
   verbs:     ['use']
   resourceNames:
-  - {{ template "external-dns.fullname" . }}
+  - {{ printf "%s-%s" (include "common.names.fullname.namespace" .) "psp" }}
```

You should be using the other form

```suggestion
  - {{ template "common.names.fullname.namespace" . }}
```

---

I was using this form initially, but it was suggested by @FraPazGal I should change it to what you see here.
Could you guys agree on what you see fit for the task?
I am totally not an expert so I am following your guidance her.e

---

The comment suggested by @FraPazGal is right, but it refers to those lines previously referencing the `-psp` suffix. Lines that did not have the suffix should not add it in this version either.

We are both stating the same thing here.

---

Now that I realised the mistake I made, I fixed it.
I added the suffix where it wasn't needed.